examples/with-sdk-js/src/app/page.tsx

```diff
@@ -1613,6 +1613,63 @@ export default function AuthPage() {
             Connect/Disconnect Wallet
           </button>
 
+          <button
+            onClick={async () => {
+              if (!activeWalletAccount) {
+                console.error("No active wallet account selected");
+                return;
+              }
+
+              await turnkey.handleSendTransaction({
+                from: activeWalletAccount.address,
+                to: activeWalletAccount.address,
+                value: "1000",
+                data: "0x",
+                caip2: "eip155:8453", // Base mainnet (example)
+                sponsor: true, // SPONSORED TX
+                successPageDuration: 3000,
+              });
+            }}
+            style={{
+              backgroundColor: "rebeccapurple",
+              borderRadius: "8px",
+              padding: "8px 16px",
+              color: "white",
+            }}
+          >
+            Send Sponsored ETH Transaction
+          </button>
+
+          <button
+            onClick={async () => {
+              if (!activeWalletAccount) {
+                console.error("No active wallet account selected");
+                return;
+              }
+
+              await turnkey.handleSendTransaction({
+                from: activeWalletAccount.address,
+                to: "0x000000000000000000000000000000000000dead",
+                value: "1", // 1 wei for example
+                data: "0x",
```

Comment:
you should be able to omit this - but maybe nice to keep for example so ppl know its an option